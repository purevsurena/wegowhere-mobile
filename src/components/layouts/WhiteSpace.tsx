import React from 'react';
import { View } from 'react-native';
import { WhiteSpaceProps } from '@/types/layouts';

function WhiteSpace(props: WhiteSpaceProps): JSX.Element {
  const { size = '12', style } = props;
  return <View style={[{ height: parseInt(size, 10) }, style]} />;
}

export default WhiteSpace;
