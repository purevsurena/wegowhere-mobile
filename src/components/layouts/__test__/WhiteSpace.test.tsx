import React from 'react';
import renderer from 'react-test-renderer';
import WhiteSpace from '@/components/layouts/WhiteSpace';

describe('<WhiteSpace />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<WhiteSpace size="12" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
