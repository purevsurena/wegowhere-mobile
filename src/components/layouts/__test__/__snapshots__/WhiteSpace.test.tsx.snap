// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WhiteSpace /> renders correctly 1`] = `
<View
  style={
    [
      {
        "height": 12,
      },
      undefined,
    ]
  }
/>
`;
